tests/test_views.py

```diff
@@ -2,6 +2,8 @@
 
 from flasql import views
 
+from tests.fixtures import app # noqa
```

Comment:
Because flake8 thinks that `app` is not being used, due to the way `app` is being injected into the test function by pytest.